types/google-apps-script/apis/calendar_v3.d.ts

```diff
@@ -361,6 +361,7 @@ declare namespace GoogleAppsScript {
         end?: Calendar.Schema.EventDateTime | undefined;
         endTimeUnspecified?: boolean | undefined;
         etag?: string | undefined;
+        eventType?: string | undefined;
```

Comment:
I think it is more appropriate:

```suggestion
        eventType?: "default" | "outOfOffice" | "focusTime" | "workingLocation";
```

Reply:
The other enum-like places were also `string`s, so I made this one a string too. But I'm going to be strict about what you suggested ❤️

Reply:
> The other enum-like places were also `string`s

That is because this type definition is old and has many contributors... 😄
Thanks a lot for your contribution 👍